uber/templates/emails/placeholders/volunteer.txt

```diff
@@ -1,6 +1,6 @@
 {{ attendee.first_name }},
 
-Thanks for volunteering to help run {{ c.EVENT_NAME }}!  You've been added to our registration database, but we don't have all of your personal information.  To ensure that you can pick up your badge with no hassles at our registration desk, please fill out the rest of your info at {{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }} and then simply bring a photo ID to {{ c.EVENT_NAME }}.
+Thanks for volunteering to help run {{ c.EVENT_NAME_AND_YEAR }}!  You've been added to our registration database, but we don't have all of your personal information.  To ensure that you can pick up your badge with no hassles at our registration desk, please fill out the rest of your info at {{ c.URL_BASE }}/preregistration/confirm?id={{ attendee.id }} and then simply bring a photo ID to {{ c.EVENT_NAME_AND_YEAR }}.
 
```

Comment:
seems like this event_name_and_year stuff got picked up in this change as well, can you revert this?

one easy way: you can actually change it by clicking the 'edit' icon in the upper right and changing it back, it should disappear in this pull request afterwards